contents.json

```diff
@@ -5464,6 +5464,11 @@
       "category": "utility",
       "description": "run swift code on slack.",
       "homepage": "https://github.com/noppefoxwolf/Swiftbot"
+    }, {
+      "title": "Real-time Chat with Firebase",
+      "category": "chat",
+      "description": "A functional real-time messaging app in Swift with Firebase Firestore.",
```

Comment:
@mrcflorian could you remove "in Swift" from description?

Reply:
Good catch, oops. Thanks!